charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -11,4 +11,16 @@ spec:
     name: machine-controller-manager
   updatePolicy:
     updateMode: {{ .Values.vpa.updatePolicy.updateMode | quote }}
+  resourcePolicy:
+    containerPolicies:
+    - containerName: machine-controller-manager-provider-equinix-metal
+      maxAllowed:
+        cpu: 2
+        memory: 5G
```

Comment:
Do these values make sense for the component owners? I'm not sure about the context this provider is used in: what's a typical machine's configuration?
Note: we're just trying to avoid that this pod becomes unschedulable for some reason, so this is meant to be some kind of rough upper scaling limit.